fuzzers/007-timing/bel/bel_suffix_map.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ff_init": ["ql", "qh"],
```

Comment:
This seems duplicate-ish with the pin_alias.  Should these be 1 file?

Reply:
It is not a duplicate. This map is intended to provide explicit suffixes of a bel name which cannot be detected using other manners.

For example, for the FF_INIT we have different SR to Q timings depending on whether we are setting Q to 0 or 1 asynchronously. This corresponds do either FDCE (`*_ql`) or FDPE (`*_qh`) primitive. So we need to have separate FF_INIT_QL and FF_INIT_QH in SDF as their counterparts. The same is for the REG_INIT_FF bel.

Now, the timings for FF_INIT are reported by Vivado as following:

```
ff_init_ce_clk
ff_init_ce_clk
ff_init_din_clk
ff_init_din_clk
ff_init_init_clk
ff_init_sr_clk_qh
ff_init_sr_clk_ql
ff_init_sr_clk_qh
ff_init_sr_clk_ql
ff_init_init_clk
ff_init_clk_q
ff_init_sr_qh
ff_init_sr_ql
```

Note that we have eg. `ff_init_sr_clk_qh` and `ff_init_sr_qh`. 

The former is a recovery/remove time for SR w.r.t. CLK for the FDPE. The latter corresponds to an async iopath timing between SR and Q. Those names refer both to the output pin (Q) and bel configuration variant (`ql` and `qh`). 

I added the alias map for pin names so we can correctly detect that both QL and QH refer to the same pin Q. But we also need to know that those names are not just pin aliases. They describe the configuration variants.

I am open to suggestions if you have a different solution to the problem in mind.

@kgugala We also had a discussion about that. You may want to look into the explanation above.

Reply:
I agree with @litghost that this is duplicating info from pin_mappings. My idea here is to use the custom pin mappings as the last resort when we were unable to find the pins. If a pin is found using this mapping we emit a timing entry using the pin name, for cell instance named with the original property.
@mkurc-ant does that make sense?